appmap.yml

```diff
@@ -0,0 +1,96 @@
+log:
+  level: info
+# Do not use node async_hooks to order callbacks.
+# This has a big performance hit.
+ordering: chronological
+# Only use safe serialization methods.
+# For instance `object.toString()` can trigger arbritary code.
+# In particular, it can cause infinite recursion.
```

Comment:
I logged an issue for this: https://github.com/getappmap/appmap-agent-js/issues/208